src/common/transformations/src/transformations/common_optimizations/transpose_sinking_pad.cpp

```diff
@@ -0,0 +1,104 @@
+#include "transformations/common_optimizations/transpose_sinking_pad.hpp"
+
+#include <openvino/pass/pattern/op/or.hpp>
+
+#include "itt.hpp"
+#include "openvino/op/util/op_types.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "openvino/util/common_util.hpp"
+#include "transformations/common_optimizations/transpose_sinking_utils.hpp"
+#include "transformations/rt_info/transpose_sinking_attr.hpp"
+
+using namespace ov::pass::pattern;
+using namespace ov;
+using namespace ov::opset10;
+using namespace transpose_sinking;
+
+ov::pass::TransposeSinkingPadForward::TransposeSinkingPadForward() {
+    MATCHER_SCOPE(TransposeSinkingPadForward);
+    auto transpose_label = wrap_type<Transpose>({any_input(), any_input()});
```

Comment:
It will be more readable. I see a pattern and realize that only constant considered as input(1). I don't need to investigate source code below to find out it.

Reply:
done